great_expectations/validator/validator.py

```diff
@@ -1511,6 +1492,39 @@ def validate(  # noqa: C901, PLR0912, PLR0913, PLR0915
 
         return result
 
+    def process_expectations_for_validation(
+        self,
+        expectation_configurations: list[ExpectationConfiguration],
+        evaluation_parameters: Optional[dict[str, Any]] = None,
+    ) -> list[ExpectationConfiguration]:
+        """Substitute evaluation parameters into the provided expectations and sort by column."""
+        NO_COLUMN = (
+            "_nocolumn"  # just used to group expectations that don't specify a column
```

Comment:
Thanks for making this a constant. It appears this magic string is defined and used in a similar way in multiple places in the codebase. Seems like it should be a defined in 1 place for all consumers of it.
This isn't necessary for this PR, just noting.